core-model-test/framework/src/main/java/org/jboss/as/core/model/test/CoreModelTestDelegate.java

```diff
@@ -200,7 +199,7 @@ private void validateDescriptionProviders(TestModelType type, KernelServices ker
             //Big big hack to get around the fact that the tests install the host description twice
             //we're only interested in the host model anyway
             //See KnownIssuesValidator.createHostPlatformMBeanAddress
-            model = model.require(CHILDREN).require(HOST).require(MODEL_DESCRIPTION).require(MASTER);
+            model = model.require(CHILDREN).require(HOST).require(MODEL_DESCRIPTION).require("master");
```

Comment:
It looks like you missed my [comment](https://github.com/wildfly/wildfly-core/pull/5108#pullrequestreview-1044858446);  this should be replaced by "primary"